### ISSUE TITLE
Add preset editor for Custom Bake Targets

### ISSUE THREAD

**Issue by trashmaster9000** [ucupumar/ucupaint#231]
i think it would be cool to have some kind of preset editor for custom bake targets

if i want to bake layers for e.g. unity i have to choose the blank preset, change RGB layers to Color, Alpha to roughness, invert alpha channel etc. which gets a bit tideous if i have to set up many objects.
it would be cool if it would be possible to create custom presets

OT:
i absolutely love your addon and would like to support your work!
is the sponsor button of github good or is there a better way? never used this button before and don't know how much github keeps for themself...

**Comment by ucupumar** [comment]
Yeah, it will be handy, but I'm still not experienced with managing add-on custom presets, so it may take me a while to implement it unless someone else does it first.

About the sponsor button, yeah it's useful for directly supporting me for the addon work, I haven't tried to take the money yet since not that many supported me yet so I'm not sure about the calculation 😅

**Comment by trashmaster9000** [comment]
no worries, hope you can get around to adding this feature some time in the future!
wish i could contribute it myself

sent you something, hope it worked!
thank you for your awesome work and happy holidays! <3

**Comment by ucupumar** [comment]
Thank you so much! 🙏🙏
I'll check again how other addons usually did their custom preset when I have the time. 

Also wish you a happy holiday!

**Comment by trashmaster9000** [comment]
just had a look around, looks like the "Modifier List" addon saves presets, maybe thats a good starting point?
https://github.com/Dangry98/modifier_list-for-Blender-4.0

it just saves a .json file to blenders Config folder (blender/[version]/config/bl_ext.blender_org.Modifier_List_Fork/preferences.json)

maybe it would be easiest to create one file per preset, in which the Image Channels are mapped to the correct render channels, then just list the files in the "New Bake target" dialogue and load it accordingly
I'm no programmer, i think you'll probably come up with something more elegant 😅

**Comment by ucupumar** [comment]
Adding a custom preset is still not implemented yet, but this PR (#256) is already merged, so at least now you can copy and paste bake target across node setups.
